services/portal/home/homeTasks.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
   login(username) {
     this.goToHomepage();
     I.setCookie({ name: 'dev_login', value: username });
+    portal.clickProp(homeProps.loginButton);
```

Comment:
this will fail if a commons is using an older portal?

Reply:
added a flag to handle portal versions on which to run the test